calcom/cal.com
# WIP form inputs for seats/slots

## What does this PR do?

Part of the event type seats feature.

This PR allows you to specify the number of seats per event type.


## Type of change



- [ x ] New feature (non-breaking change which adds functionality)


## How should this be tested?

- Go to edit an event type and show additional options
- At the bottom click the check box to offer seats
    - Ensure that disable guests is enabled
- Choose an option on the drop down menu that is not "6+"
- Save the event, you should see the seat numbers reflected in the db under `eventTypes` ->  `seatsPerTimeSlot`
- Go back to the event type and choose the "6+" option
    - Need to be in a pro account
- Enter a number in the new input and hit save
- The `seatsPerTimeSlot` should be updated
- In the event type uncheck the option to offer seats
- Save and in the db `seatsPerTimeSlot` should now be null

## Checklist



- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
